<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosticsTab</class>
 <widget class="QWidget" name="DiagnosticsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <item row="1" column="0">
    <widget class="QGroupBox" name="deviceGroupBox">
     <property name="title">
      <string>Device Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="halVersionLabel">
        <property name="text">
         <string>HAL Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mcuDevIdLabel">
        <property name="text">
         <string>MCU Device ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mcuRevIdLabel">
        <property name="text">
         <string>MCU Revision ID:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mcuSysClockLabel">
        <property name="text">
         <string>MCU SysClock:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="uniqueIdLabel">
        <property name="text">
         <string>Unique ID:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mcuVddaLabel">
        <property name="text">
         <string>Vdda:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="mcuTempLabel">
        <property name="text">
         <string>MCU Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="sensorTempLabel">
        <property name="text">
         <string>Sensor Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="halVersionLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mcuDevIdLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mcuRevIdLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mcuSysClockLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="uniqueIdLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="mcuVddaLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="mcuTempLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="sensorTempLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="commandGroupBox">
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="screenshotButton">
        <property name="text">
         <string>Take Screenshot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshSensorsPushButton">
        <property name="text">
         <string>Refresh Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remotePushButton">
        <property name="text">
         <string>Remote Control</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="firmwareGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Firmware Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="versionLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="buildDateLabel">
        <property name="text">
         <string>Build Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="buildDateLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="buildCommitLabel">
        <property name="text">
         <string>Commit:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="buildCommitLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="buildChecksumLabel">
        <property name="text">
         <string>Checksum:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="buildChecksumLineEdit">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string notr="true">&lt;Device Name&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>versionLineEdit</tabstop>
  <tabstop>buildDateLineEdit</tabstop>
  <tabstop>buildCommitLineEdit</tabstop>
  <tabstop>buildChecksumLineEdit</tabstop>
  <tabstop>halVersionLineEdit</tabstop>
  <tabstop>mcuDevIdLineEdit</tabstop>
  <tabstop>mcuRevIdLineEdit</tabstop>
  <tabstop>mcuSysClockLineEdit</tabstop>
  <tabstop>uniqueIdLineEdit</tabstop>
  <tabstop>mcuVddaLineEdit</tabstop>
  <tabstop>mcuTempLineEdit</tabstop>
  <tabstop>sensorTempLineEdit</tabstop>
  <tabstop>screenshotButton</tabstop>
  <tabstop>refreshSensorsPushButton</tabstop>
  <tabstop>remotePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
